micro-rdk-installer/README.md

```diff
@@ -2,10 +2,16 @@
 
 A CLI that allows a user to flash a build of Micro-RDK, along with their robot's credentials and their wifi information, directly to their esp32 without requiring installation of ESP-IDF, Rust, or Python.
 
+## Download
+
+Find the compiled installer executable corresponding to your system's architecture under the [latest release](https://github.com/viamrobotics/micro-rdk/releases/latest) of micro-RDK.
+
 ## Build Instructions
 
+Only necessary as an alternative to the previous Download step 
```

Comment:
Confused on what they're building/where this starts if they haven't downloaded anything yet. How does that work?